src/components/Capture/capture.js

```diff
@@ -97,11 +98,15 @@ class Capture extends Component {
       return
     }
     payload = {...payload, documentType}
-    if (this.props.side === 'back' && !this.props.useWebcam) {
-      payload = {...payload, valid: true}
+    if (this.props.useWebcam) {
+      postToServer(this.createJSONPayload(payload), serverUrl, token,
+        (response) => this.onServerResponse(response),
```

Comment:
No, the `valid` property is only returned by the SDK backend and `onServerResponse` is only called when we receive a response from the server (SDK backend).
I don't return the document/face id because in order to run a document or facial similarity check the client doesn't need the doc/face id, and I agreed about this with See Wah during feature design. At the same time I agree with you on the fact that we are already including the id returned by the SDK backend in the documentCapture object and for consistency we could just replace it with the document id returned by the api. In that case, is it worth checking with Richard about potential security risks in returning the document id on the client? Anyways, I think this should be handled as part of another story because this isn't a blocker and I want to prioritise finishing the error feature for webcam screen during this sprint.